akka-persistence/src/main/scala/akka/persistence/Eventsourced.scala

```diff
@@ -254,8 +254,12 @@ private[persistence] trait Eventsourced extends Snapshotter with Stash with Stas
    * @param events events to be persisted
    * @param handler handler for each persisted `events`
    */
+  final def persistAll[A](events: immutable.Seq[A])(handler: A ⇒ Unit): Unit =
+    events.foreach(persist(_)(handler)) // TODO the atomic part should be handled by issue #15377
+
+  @deprecated("use persistAll instead", "2.4")
   final def persist[A](events: immutable.Seq[A])(handler: A ⇒ Unit): Unit =
-    events.foreach(persist(_)(handler))
+    persistAll(events)(handler)
```

Comment:
That's very true - I hope most users IDEs can properly figure out the deprecated one is used and highlight it in the editor. So the plan would be to keep it in during the Milestone phase and then drop in RC? Let's create a ticket for it so we don't forget (I for one have terrible memory for these kinds of things :wink:)